.github/workflows/automation-trigger-test.yml

```diff
@@ -1,18 +1,101 @@
-#on: [push]
+name: appium-automation
 on:
- workflow_dispatch:
- 
+  workflow_dispatch:
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+env:
+  ACCESS_KEY: ${{ secrets.S3_ACCESS_KEY }}
+  SECRET_KEY: ${{ secrets.S3_SECRET_KEY }}
+  BROWSERSTACK_USER: ${{ secrets.BROWSER_STACK_USER }}
+  BROWSERSTACK_KEY: ${{ secrets.BROWSER_STACK_KEY }}
+  IS_RN_APP: true
 jobs:
-  build:
-    runs-on: ubuntu-latest
+  appium-automation-tests:
+    name: appium-automation-tests
+    runs-on: macos-latest
     steps:
-    - uses: actions/github-script@v6
-      with:
-        github-token: ${{ secrets.BRANCHLET_ACCESS_TOKEN_PUBLIC  }}
-        script: |
-          await github.rest.actions.createWorkflowDispatch({
-            owner: 'BranchMetrics',
-            repo: 'qentelli-saas-sdk-react-native-testing-automation',
-            workflow_id: 'BrowserStack_ReactNative_Tests.yml',
-            ref: 'master'
-          })
+      - name: Checkout SDK repo with testbed app
+        uses: actions/checkout@v3
+        with:
+          path: sdk
+          ref: SDK-1972-move-qentelli-automation-testbed
+      # point to automation master branch by default
+      - name: Checkout qa automation
+        uses: actions/checkout@v3
+        with:
+          path: qa
+          repository: BranchMetrics/qentelli-saas-sdk-testing-automation
+          token: ${{ secrets.BRANCHLET_ACCESS_TOKEN_PUBLIC }}
+          ref: SDK-1984-Merge-ReactNativeRepo
```

Comment:
I approved the automation side, once this is pointed to master i will approve this.

Reply:
GHAs look good to me